src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/entities/NoteResource.java

```diff
@@ -30,7 +31,11 @@ private NoteResource() {
     }
 
     @JsonCreator
-    public NoteResource(String code, String noteType, String user, OffsetDateTime created, String content) {
+    public NoteResource(@JsonProperty("code") String code,
+                        @JsonProperty("noteType") String noteType,
+                        @JsonProperty("user") String user,
+                        @JsonProperty("created") OffsetDateTime created,
+                        @JsonProperty("content") String content) {
```

Comment:
Let's try to avoid the JsonProperty especially in this case it makes it camel case our APIs should only be using snake case